src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -925,6 +925,7 @@ private function registerTranslatorConfiguration(array $config, ContainerBuilder
 
         // Use the "real" translator instead of the identity default
         $container->setAlias('translator', 'translator.default');
+        $container->setAlias('translator.formatter', $config['formatter']);
```

Comment:
:-1: for FQCN here, that'd be inconsistent
but the alias should be private, isn't it?
about the MessageFormatterInterface alias, that'd not for this PR since that'd be a "new feature".
but we did not add this alias in 3.3. Is there any real use case for type hinting MessageFormatterInterface and thus have it autowired?